example/index.js

```diff
@@ -1,19 +1,28 @@
+import Web3 from "web3"
+import web3Modal from './providers.js'
+
+let web3
+
+const getProvider = async () => {
+  const provider = await web3Modal.connect();
```

Comment:
It really sucks that you have to do this first in order to get the provider. Now we can't create 3box on pageload again. Should be easy to add a change where you can just a pass the provider in the auth stage! (which I think makes more sense anyway).

Reply:
yeah this was true before for everyone, but yeah agree, wasnt going to do for this release, but easy enough to add as option anyways